projects/woff2/build.sh

```diff
@@ -0,0 +1,109 @@
+# Copyright 2024 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+#!/bin/bash
+
+export CC=clang
+export CXX=clang++
+export SRC=.
+
+# Build the library. Actually there is no 'library' target, so we use .o files.
+# '-no-canonical-prefixes' flag makes clang crazy. Need to avoid it.
+cat brotli/shared.mk | sed -e "s/-no-canonical-prefixes//" > brotli/shared.mk.temp
+mv brotli/shared.mk.temp brotli/shared.mk
+
+# Build targets for fuzzing
+export CFLAGS="-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
+export CXXFLAGS="-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
+
+make clean
+make CC="$CC $CFLAGS" CXX="$CXX $CXXFLAGS" CANONICAL_PREFIXES= all NOISY_LOGGING=
+
+export CFLAGS="-g -fsanitize=fuzzer,address,integer,bounds,null,undefined,float-divide-by-zero"
+export CXXFLAGS="-g -fsanitize=fuzzer,address,integer,bounds,null,undefined,float-divide-by-zero"
+
+for fuzzer_archive in ./src/*fuzzer*.a; do
+  fuzzer_name=$(basename ${fuzzer_archive%.a})
+  $CXX $CXXFLAGS ./${fuzzer_name}.cc $fuzzer_archive -I ./src -I ./include -I ./brotli/c/dec -I ./brotli/c/common \
```

Comment:
All source files for fuzzers are located in `src` directory of the project repo. So we can just specify the appropriate path to them inside the repo and remove them from our repo (we needn't keep them on our side). Here and later